pom.xml

```diff
@@ -19,9 +19,8 @@
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter</artifactId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
let's create separate dependency for web and return the original dependency, that you changed